sdk/appcenter/src/test/java/com/microsoft/appcenter/AppCenterAuthTest.java

```diff
@@ -42,6 +42,44 @@ public void setUp() throws Exception {
         when(AuthTokenContext.getInstance()).thenReturn(mAuthTokenContext);
     }
 
+    @Test
+    public void setAuthToken() {
+
+        /* Given a valid JWT token. */
+        final String jwt = "jwt";
+        JwtClaims claims = mock(JwtClaims.class);
+        when(claims.getSubject()).thenReturn("someId");
+        when(claims.getExpirationDate()).thenReturn(new Date(123L));
+        when(JwtClaims.parse(jwt)).thenReturn(claims);
+
+        /* When we set auth token. */
+        AppCenter.setAuthToken(jwt);
+
+        /* Then it's stored. */
+        verify(mAuthTokenContext).setAuthToken(jwt, claims.getSubject(), claims.getExpirationDate());
+    }
+
+    @Test
+    public void unsetAuthToken() {
+
+        /* When we unset auth token. */
+        AppCenter.setAuthToken(null);
+
+        /* Then it's removed. */
+        verify(mAuthTokenContext).setAuthToken(null, null, null);
```

Comment:
And `JwtClaims.parse` isn't called.

Reply:
I think it is fine but we can also verify that isn't called.